### ISSUE TITLE
[Feature] composite tasks not visible in workflow UI page before running the workflow

### ISSUE THREAD

**Issue by kosigz-lyft** [lyft/flyte#613]
**Motivation: Why do you think this is important?**
While using the Flyte UI, I found that you cannot see the tasks within a workflow without running the workflow. This can be confusing in the case where a developer adds a new task, but is not sure if it's been picked up correctly before running the workflow.

**Goal: What should the final outcome look like, ideally?**
Ideally, when one opens the Flyte console, they should be able to see the tasks that are part of a workflow without running the workflow. Currently, all you can see are runs and you have to have a started run to see the actual execution graph.

**Describe alternatives you've considered**
The workaround involves running the workflow to see the tasks.

**Flyte component**
- [ ] Overall
- [ ] Flyte Setup and Installation scripts
- [ ] Flyte Documentation
- [ ] Flyte communication (slack/email etc)
- [ ] FlytePropeller
- [ ] FlyteIDL (Flyte specification language)
- [x] Flytekit (Python SDK)
- [ ] FlyteAdmin (Control Plane service)
- [ ] FlytePlugins
- [ ] DataCatalog 
- [ ] FlyteStdlib (common libraries)
- [x] FlyteConsole (UI)
- [ ] Other

**Additional context**
n/a

**Is this a blocker for you to adopt Flyte**
n/a already using Flyte.

**Comment by kumare3** [comment]
@kosigz-lyft i agree with your comment and indeed this was the initial feature. But based on some user feedback we put executions front and center. Cc @schottra do we have UX plans to show versions of workflows that allows users to examine the structure?

@kosigz-lyft I appreciate the feedback and we will as a team explore the right ic around this request

**Comment by schottra** [comment]
@kumare3 We don't currently have a planned item to expose workflow versions again. But having a use case helps drive it.
I think it makes sense to be able to view the individual versions of a workflow and see a graph/task details. So let's keep this issue open to track it and we can figure out when to prioritize it.
I think it will need at least a lightweight spec document beforehand though.
